charts/metallb/charts/crds/templates/crds.yaml

```diff
@@ -26,7 +26,7 @@ spec:
         caBundle: Cg==
         service:
           namespace: {{ .Release.Namespace }}
-          name: {{ template "metallb.fullname" . }}-webhook-service
```

Comment:
Now that these are in a sub-chart, they should bue fully templatable.

Reply:
but the name will be different between the subchart and the main chart, it is adding both names here to the prefix - metallb-metallb-crds (i wrote that in my commit message)

Reply:
and anyway, why do we need the webhook service name templated?

Reply:
I think it's ok to un-template the service name. Why do you think it will be incorrect if the user is using another name for the release? if we un-template that and use the same service name all over the manifests I think we are good. Am I missing something here?

Reply:
If a user uses a non-standard release name, (say `foo`) I'm fairly certain the templated service name will come out as `foo-metallb-webhook-service` because of this block:
https://github.com/metallb/metallb/blob/10df3fa158dd99b0ba687b625d49129a6483e03b/charts/metallb/templates/_helpers.tpl#L19-L23

By my read the above means "if the release name contains the chart name, render the fullname as the release name, else render the fullname as releasename-chartname".

This will be different than the non-templated name `metallb-webhook-service` defined in the CRD for conversion, and will cause conversion to fail. 

@fedepaol would you mind double-checking my logic here? Relevant docs for the helm `contains` function in that code block [here](https://helm.sh/docs/chart_template_guide/function_list/#contains).

Reply:
Spoke with @gclawes , I think we can move forward with the current status (which is, the service name is fixed).
Having multiple instance of metallb using different names, especially in the same namespace seems non realistic so it should not be a problem.